src/driver/postgres/PostgresDriver.ts

```diff
@@ -556,6 +573,17 @@ export class PostgresDriver implements Driver {
                 )
             },
         )
+        const hasVectorColumns = this.connection.entityMetadatas.some(
+            (metadata) => {
+                return (
+                    metadata.columns.filter(
```

Comment:
`Array.some` also works and doesn't create a new array :)

Reply:
Great point, thanks!